Multidb support Take 2 (#1)
flea89/django-mass-migration: README.md

These can still be tracked the same as the other operations, but the implementation of what your migration
does and how your the backend handles it are up to you.

Running with Multiple DBs
-------------------

> If your projects connects to multiple databases it is possible to run your migrations against all of them singularly.

Reviewer: ```suggestion
If your project connects to multiple databases it is possible to run your migrations against each database individually.
```
